cluster-operator/src/main/resources/config-maps/050-ConfigMap-strimzi-cluster-operator-logging.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: strimzi-cluster-operator-logging
+data:
+  log4j2.properties: |
+    name = COConfig
+
+    appender.console.type = Console
+    appender.console.name = STDOUT
+    appender.console.layout.type = PatternLayout
+    appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+    rootLogger.level = ${env:STRIMZI_LOG_LEVEL:-INFO}
+    rootLogger.appenderRefs = stdout
+    rootLogger.appenderRef.console.ref = STDOUT
+    rootLogger.additivity = false
+
+    # Kafka AdminClient logging is a bit noisy at INFO level
+    logger.kafka.name = org.apache.kafka
+    logger.kafka.level = ${env:STRIMZI_AC_LOG_LEVEL:-WARN}
+    logger.kafka.additivity = false
+
+    # Zookeeper is very verbose even on INFO level -> We set it to WARN by default
+    logger.zookeepertrustmanager.name = org.apache.zookeeper
+    logger.zookeepertrustmanager.level = ${env:STRIMZI_ZOOKEEPER_LOG_LEVEL:-WARN}
```

Comment:
So, why does this still use the env vars? It should just have the log levels, or?

Reply:
All the envvars are kept for compatibility with older versions of deployment file. Isn't it something we want?

Reply:
I do not understand what do you mean with compatibility. To get the config from the configmap you need to apply the changes and use new operator basically. So what kind of compatibility are we talking about?

Reply:
If users wants to use envvars to set the logging, this allows it. Of course with rolling update.